packages/client/hmi-client/src/components/home/tera-project-card.vue

```diff
@@ -1,27 +1,37 @@
 <template>
 	<Card v-if="project">
 		<template #content>
-			<header class="project-stats">
-				<span title="Contributors"><i class="pi pi-user" /> {{ stats?.contributors }}</span>
-				<span title="Models"><i class="pi pi-share-alt" /> {{ stats?.models }}</span>
+			<header>
+				<span title="Contributors"><i class="pi pi-user" /> {{ stats?.contributors }}</span
+				><span title="Papers"><i class="pi pi-file" /> {{ stats?.papers }}</span>
 				<span title="Datasets"
 					><dataset-icon fill="var(--text-color-secondary)" /> {{ stats?.datasets }}</span
-				>
-				<span title="Papers"><i class="pi pi-file" /> {{ stats?.papers }}</span>
+				><span title="Models"><i class="pi pi-share-alt" /> {{ stats?.models }}</span>
 			</header>
-			<div class="project-img">
+			<div class="img">
 				<img :src="image" alt="Artistic representation of the Project statistics" />
 			</div>
-			<div class="project-title">{{ project.name }}</div>
-			<div class="project-description">{{ project.description }}</div>
-			<div class="project-footer">
-				<span>Last updated {{ formatDdMmmYyyy(project.timestamp) }}</span>
-				<Button
-					icon="pi pi-ellipsis-v"
-					class="p-button-icon-only p-button-text p-button-rounded"
-					@click.stop="showProjectMenu"
-				/>
-			</div>
+			<section>
+				<div class="title" ref="titleRef">
+					{{ project.name }}
+				</div>
+				<section class="details">
+					<div>
+						<div class="author">{{ project.username }}</div>
+						<div class="creation-date">{{ formatDdMmmYyyy(project.timestamp) }}</div>
+					</div>
+					<div class="description">{{ project.description }}</div>
+				</section>
+			</section>
+		</template>
+		<template #footer>
+			<!--FIXME: There is no 'last updated' property in project yet-->
+			<span>Last updated {{ formatDdMmmYyyy(project.timestamp) }}</span>
+			<Button
+				icon="pi pi-ellipsis-v"
+				class="p-button-icon-only p-button-text p-button-rounded"
+				@click.stop="showProjectMenu"
+			/>
```

Comment:
Maybe because this button is not visible until on hover that its padding/margin are changing the footer and make it jump.

Reply:
That wasn't it, I think the `-webkit-box` triggers some weird behaviour. I ended up giving the footer an absolute position relative to the card body.